client/dhcp_client.ml

```diff
@@ -124,6 +141,7 @@ let create ?with_xid ?requests srcmac =
     chaddr = srcmac;
     options = [
       Message_type DHCPDISCOVER;
+      Client_id (Id (Macaddr.to_string srcmac));
```

Comment:
This is my fault as I made the original suggestion (sorry!) but the `Client_id` variant allows for specifying the MAC address directly, as:

```
Client_id (Hwaddr srcmac)
```

rather than serializing the MAC and then passing the string, and I think that would be better.

Reply:
No problem, changed to specify the MAC address directly. Thanks!